scripts/test

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${CI}" ]]; then
+    set -x
+fi
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Execute project linters and test suites. Assumes that the execution environment
+has `requirements-dev.txt` installed.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        # Lint
+        flake8 pystac_io
+
+        # Code formatting
+        black pystac_io
```

Comment:
should this _do_ formatting or _check_ formatting?

Reply:
Good catch. This should have a `--check`.